### ISSUE TITLE
Remove motion log translations from REST data.

### ISSUE THREAD

**Issue by normanjaeckel** [OpenSlides/OpenSlides#2025]
See #2018: Translations checked in /rest/: Only motion logs are important. Config labels comes with option request (not via auto update). All other apps (motions/assingments/mediafiles/agenda) have no special translations in REST and auto update data. So we only have to fix the motion log translation after auto update (that not the browser language is used from user who triggers the auto update). Therefore we send only English strings and translate them on client side.

**Comment by emanuelschuetze** [comment]
How to reproduce this issue?
* reload with German browser
* switch OpenSlides client language to English
* create a motion and edit it afterwards
* go to the motion detail view and show there the log/history
-> result: the content of the log is in German instead of English.

**Comment by emanuelschuetze** [comment]
@normanjaeckel What is to do here on server side? Can you summarize it that an other developer can understand and implement it?

**Comment by normanjaeckel** [comment]
First try to reproduce the following:
* Open one Session (user 1) with a German browser.
* Open anonther Session (user 2) with an English browser,
* Create an edit a motion with both users.

Now the motion log should contain different languages.

If you get this then the problem is the browser language set during autoupdate. We have to remove all translation flag on server side to force the server to send only English strings. The the client should add translation for this data like for permissions, config, ...

**Comment by matakuka** [comment]
I don't understand the difference between `gettext` aka `_` and `gettext_noop` (which is used extensively in the log). A quick googling wasn't very helpful.

**Comment by normanjaeckel** [comment]
`ugettext` aka `_` means that Python tries to translate this string during runtime. `ugettext_noop` means something like “no operation”. The string is marked as translatable so that the translation tools can find and insert it into the po files. But during runtime the string is not translated. The developer has to care that the variable or object comes somewhere else to a place where it is translated. For motion log it is motions/serializer.py line 91 which refers to motion/models line 682 and 689.

**Comment by normanjaeckel** [comment]
https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#marking-strings-as-no-op

**Comment by jsaalfeld** [comment]
Since the client will be rewritten for OS3, this shouldn't be an issue anymore. Please reopen if otherwise.

**Comment by tsiegleauq** [comment]
Motion log does not exist anymore. If the current issue was not yet resolved, remove the leftover parts of motions log. Otherwise close this issue.
